vpdq/python/vpdq.pyx

```diff
@@ -144,6 +144,8 @@ def computeHash(
 
     if downsample_height == 0:
         downsample_height = vid.get(cv2.CAP_PROP_FRAME_HEIGHT)
+    
```

Comment:
L141 is the source of our 0 frames_per_sec, I assume that is from ffmpeg

Reply:
The Python bindings get this information here from OpenCV, but for cpp it gets this in `vpdqio.cpp` from readVideoStreamInfo(). It's unclear to me why the Python bindings don't reuse readVideoStreamInfo(), which would avoid importing OpenCV entirely.

OpenCV has issues with some video types, even common ones like [AV1](https://github.com/opencv/opencv/issues/22304). [PyAV](https://pypi.org/project/av/) offers excellent bindings to FFmpeg for this kind of thing.